packages/osd-opensearch/src/artifact.test.js

```diff
@@ -40,26 +40,34 @@ import { Artifact } from './artifact';
 const log = new ToolingLog();
 let MOCKS;
 
+const DAILY_SNAPSHOT_BASE_URL = 'https://artifacts.opensearch.org/snapshots/core/opensearch';
+// eslint-disable-next-line no-unused-vars
+const PERMANENT_SNAPSHOT_BASE_URL = '';
+
+const ORIGINAL_PLATFROM = process.platform;
+const ORIGINAL_ARCHITECTURE = process.arch;
 const PLATFORM = process.platform === 'win32' ? 'windows' : process.platform;
 const ARCHITECTURE = process.arch === 'arm64' ? 'arm64' : 'x64';
 const MOCK_VERSION = 'test-version';
-const MOCK_URL = 'http://127.0.0.1:12345';
-const MOCK_FILENAME = 'test-filename';
+const MOCK_RC_VERSION = `test-version-rc4`;
+const MOCK_FILENAME = 'opensearch-test-version-SNAPSHOT-linux-x64-latest.tar.gz';
+const MOCK_RC_FILENAME = `opensearch-test-version-rc4-SNAPSHOT-linux-x64-latest.tar.gz`;
+const MOCK_URL = `${DAILY_SNAPSHOT_BASE_URL}/${MOCK_VERSION}/${MOCK_FILENAME}`;
+const MOCK_RC_URL = `${DAILY_SNAPSHOT_BASE_URL}/${MOCK_RC_VERSION}/${MOCK_RC_FILENAME}`;
 
-const DAILY_SNAPSHOT_BASE_URL = '';
-const PERMANENT_SNAPSHOT_BASE_URL = '';
+const itif = process.platfom !== 'linux' || process.arch !== 'x64' ? it : it.skip;
```

Comment:
So we only skip the tests if we're running on Linux x64? If that's the case why not write the inverse? I know we used this logic above because snapshots are only available for that combination but I think this would be easier to read in this case:

```javascript
const itif = (process.platform === 'linux' && process.arch === 'x64') ? it.skip : it;
```

Reply:
the logic was incorrect and I think I got away with it because of the typo in platform. Some of the tests should only run on LInux x64. Since right now there only Linux x64 snapshots.